### ISSUE TITLE
Unable to change the bind port

### ISSUE THREAD

**Issue by idubinets** [cprobert/kiss-ssg#1]
Hi Team, how to change the bind port? I am getting the following error:

```
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (net.js:1316:16)
    at listenInCluster (net.js:1364:12)
    at Server.listen (net.js:1450:7)
    at Function.listen (/msc/node_modules/connect/index.js:217:24)
    at module.exports (/msc/node_modules/kiss-ssg/kiss-serve.js:31:7)
    at new Kiss (/msc/node_modules/kiss-ssg/kiss-ssg.js:367:9)
    at Object.<anonymous> (/msc/generate.js:54:12)
    at Module._compile (internal/modules/cjs/loader.js:1236:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1257:10)
    at Module.load (internal/modules/cjs/loader.js:1085:32)
```

**Comment by idubinets** [comment]
It seems it is not possible now: https://github.com/cprobert/kiss-ssg/blob/main/kiss-serve.js#L31

can you please make it configurable?

**Comment by cprobert** [comment]
No problem

I just published 1.4.4 with this as an option? See examples\5-helpers.js
for an example usage


Court

**Comment by idubinets** [comment]
thank you, works perfectly
